### PR TITLE
feat(curriculum): add animations to block 1.3

### PR DESCRIPTION
**Pull request by DanielRosa74**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dc2fb63bc6c2b596dd6e4.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 id: 657dc2fb63bc6c2b596dd6e4
-title: Task 91
+title: Task 89
 challengeType: 22
-dashedName: task-91
-audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+dashedName: task-89
 ---
 
 <!-- (audio) Sophie: Not too long. They usually take about 30 to 45 minutes.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dc2fb63bc6c2b596dd6e4.md:8]: Can you leave in the animation just Tom's line and remove Sophie?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dc6a58ec0be432655eaf5.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dc6a58ec0be432655eaf5.md:1]: Can you please update the instructions in line 23?
It should be: Listen to the audio, understand the context and complete this sentence with the 
correct adverb of frequency.

**Review by nieldakarla — COMMENTED** [review]
This looks great, Dani! I've taken the opportunity and asked a couple adjustments in some challenges. But the animations look great!

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Thanks for the changes, Daniel! LGTM

**Review by nieldakarla — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**